### ISSUE TITLE
widget:show_engines errors due to missing widget engine property

### ISSUE THREAD

**Issue by iturgeon** [ucfopen/Materia#1393]
Due to migration 48:

https://github.com/ucfopen/Materia/blob/master/fuel/app/migrations/048_remove_group_from_widget.php#L9-L12

The task will error on

https://github.com/ucfopen/Materia/blob/master/fuel/app/tasks/widget.php#L88

when attempting to print out the names of the installed engines

**Comment by iturgeon** [comment]
Output of the script when it's erroring

```
› ./run.sh php oil r widget:show_engines                                                                                                                                                                          21:21:07
[+] Running 3/0
 ⠿ Container docker-fakes3-1     Running                                                                                                                                                                              0.0s
 ⠿ Container docker-memcached-1  Running                                                                                                                                                                              0.0s
 ⠿ Container docker-mysql-1      Running                                                                                                                                                                              0.0s
wait-for-it.sh: waiting 20 seconds for mysql:3306
wait-for-it.sh: mysql:3306 is available after 0 seconds
ERROR - 2022-08-22T15:21:16-0400 --> 8 - Undefined index: group in /var/www/html/fuel/app/tasks/widget.php on line 88

Uncaught exception Fuel\Core\PhpErrorException: 8 - Undefined index: group in /var/www/html/fuel/app/tasks/widget.php on line 88
Callstack: 
#0 /var/www/html/fuel/core/bootstrap.php(112): Fuel\Core\Errorhandler::error_handler()
#1 /var/www/html/fuel/app/tasks/widget.php(88): {closure}()
#2 /var/www/html/fuel/core/base56.php(37): Fuel\Tasks\Widget::show_engines()
#3 /var/www/html/fuel/packages/oil/classes/refine.php(106): call_fuel_func_array()
#4 [internal function]: Oil\Refine::run()
#5 /var/www/html/fuel/packages/oil/classes/command.php(124): call_user_func()
#6 /var/www/html/oil(68): Oil\Command::init()
#7 {main}
```

**Comment by clpetersonucf** [comment]
Resolved in #1394
